versioned_docs/version-V2/concepts/11-governance/01-overview.md

```diff
@@ -36,7 +36,7 @@ import Timelock from "@uniswap/governance/build/Timelock.json";
 
 [https://unpkg.com/@uniswap/governance@1.0.2/build/Timelock.json](https://unpkg.com/@uniswap/governance@1.0.2/build/Timelock.json)
 
-# GovernorAlpha
+# GovernorAlpha (Deprecated)
```

Comment:
we should also reference `0xC4e172459f1E7939D522503B81AFAaC1014CE6F6` (https://etherscan.io/address/0xC4e172459f1E7939D522503B81AFAaC1014CE6F6) which was the second GovernorAlpha, also now deprecated. we should only link that one on mainnet though